lib/charms/prometheus_k8s/v0/prometheus_remote_write.py

```diff
@@ -23,7 +23,14 @@
 
 import yaml
 from charms.observability_libs.v0.juju_topology import JujuTopology
-from ops.charm import CharmBase, HookEvent, RelationEvent, RelationMeta, RelationRole
+from ops.charm import (
+    CharmBase,
+    HookEvent,
+    RelationBrokenEvent,
+    RelationEvent,
+    RelationMeta,
+    RelationRole,
+)
 from ops.framework import EventBase, EventSource, Object, ObjectEvents
 from ops.model import Relation
```

Comment:
Bump LIBPATCH?

Reply:
let's do in another PR once ready to release. Do not want to rerun CI only for this :D